packages/storybook/src/nimble/table-column/menu-button/table-column-menu-button-opened-matrix.stories.ts

```diff
@@ -0,0 +1,85 @@
+import type { StoryFn, Meta } from '@storybook/html';
+import { html } from '@microsoft/fast-element';
+import { waitForUpdatesAsync } from '@ni/nimble-components/src/testing/async-helpers';
+import { menuTag } from '@ni/nimble-components/src/menu';
+import { menuItemTag } from '@ni/nimble-components/src/menu-item';
+import { TablePageObject } from '@ni/nimble-components/src/table/testing/table.pageobject';
```

Comment:
We shouldn't be mixing import path formats, i.e. some directly to paths on disk and some via this format. Lets stick with direct import paths because we would not expect to ship typescript `src` from nimble directly via the package.

Also can you add this format to the block list (`@ni/nimble-components/*` and `@ni/spright-components/*`):

https://github.com/ni/nimble/blob/7450464c52f8dabfcac1f0fbc963a91dc23d4c4d/packages/storybook/.eslintrc.js#L52-L57